pkg/apis/policy/types.go

```diff
@@ -444,6 +449,25 @@ const (
 	SupplementalGroupsStrategyRunAsAny SupplementalGroupsStrategyType = "RunAsAny"
 )
 
+// RuntimeClassStrategyOptions define the strategy that will dictate the allowable RuntimeClasses
+// for a pod.
+type RuntimeClassStrategyOptions struct {
+	// AllowedRuntimeClassNames is a whitelist of RuntimeClass names that may be specified on a pod.
+	// A value of "*" means that any RuntimeClass name is allowed, and must be the only item in the
+	// list. An empty list requires the RuntimeClassName field to be unset.
+	AllowedRuntimeClassNames []string
+	// DefaultRuntimeClassName is the default RuntimeClassName to set on the pod.
+	// The default MUST be allowed by the above AllowedRuntimeClasses.
```

Comment:
> must be allowed by the "allowedRuntimeClassNames" list

Reply:
Is there guidance on whether references to other API fields should use the CamelCase or camelCase name? It's pretty inconsistent, but CamelCase seems more common.

Reply:
actually, all the field names should be their serialized names, since that's what is used to generate API doc

Reply:
Fixed, along with capitalization.